ospf6d/ospf6_asbr.c

```diff
@@ -1092,7 +1103,7 @@ void ospf6_asbr_routemap_update(const char *mapname)
 								type));
 
 				route_map_counter_increment(ROUTEMAP(red));
-
+				SET_FLAG(red->flag, OSPF6_IS_RMAP_CHANGED);
```

Comment:
The flag is set every time `ospf6_asbr_distribute_list_update` is called so let's move it inside the function itself.

Reply:
Fixed